tests/cli/test_suite.py

```diff
@@ -2422,25 +2422,21 @@ def test_suite_list_with_multiple_suites(
     mock_emit, caplog, monkeypatch, empty_data_context_stats_enabled
 ):
     context = empty_data_context_stats_enabled
-    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    monkeypatch.chdir(os.path.dirname(context.root_directory))  # noqa: PTH120
 
     project_dir: str = context.root_directory
 
     # noinspection PyUnusedLocal
-    suite_0: ExpectationSuite = context.add_expectation_suite(
```

Comment:
Do we not want to save the suite returned from add_expectation_suite in these 2 places?